### PR TITLE
[TASK] xdebug.max_nesting_level

### PR DESCRIPTION
**Pull request by lewisvoncken**
- [x] I have followed the [guidelines for contributing](https://github.com/weprovide/valet-plus/blob/master/CONTRIBUTING.md).
- [x] I have checked that there aren't other open [pull requests](https://github.com/weprovide/valet-plus/pulls) for the same issue ticket.
- [x] I have formatted my code with the [PSR-2](http://www.php-fig.org/psr/psr-2/) coding style before submitting my PR.
-----

**I have read the contribution guidelines and am targeting the branch `<YOUR_TARGET_BRANCH>`:**  
Because this is a Feature which is Backwards Compatible.  

**Changelog entry:**  
Short description of your work as explained by: [Keep A Changelog](https://keepachangelog.com).
### Added
- set xdebug.max_nesting_level to -1

### CODE REVIEW

File: cli/stubs/z-performance.ini [hunk 1]
```diff
@@ -67,3 +67,4 @@ xdebug.remote_host=localhost
 xdebug.remote_port=9000
 xdebug.remote_autostart=1
 xdebug.idekey=PHPSTORM
+xdebug.max_nesting_level=2000
```
Comment (helveticadomes) [inline cli/stubs/z-performance.ini:70]: at the comment you wrote 200, but here you set it to 2000

Comment (mischabraam) [reply cli/stubs/z-performance.ini]: I don't mind just setting this to `-1`.

Comment (lewisvoncken) [reply cli/stubs/z-performance.ini]: @mischabraam that has been updated.

**Comment by Neodork** [comment]
All tests green, this is getting merged!

**Pull request merged**